src/main/java/ch/njol/skript/expressions/ExprItemThrower.java

```diff
@@ -0,0 +1,79 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.bukkitutil.UUIDUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Item Thrower")
+@Description({
+	"The entity that threw/dropped the dropped item.",
+	"Getting the item thrower will only return a living entity or a player that has played before. "
+	    + "If the entity was killed, or the player has never played before, this will return null."
```

Comment:
```suggestion
		+ "If the entity was killed, or the player has never played before, this will return null."
```
Replace spaces with tabs